src/store/CategoricalDisplayStore.js

```diff
@@ -197,6 +197,15 @@ export default class CategoricalDisplayStore {
         }, 0);
     }
 
+    @action.bound clearFilterText() {
+        this.filterText = '';
+        this.searchInput.value = '';
```

Comment:
I think either only `this.filterText = '';` or `this.searchInput.value = '';` is needed. Check back with me and see?

Reply:
one of them responsible to clear input text, and another one to reset filter. if we remove any of them, the search and it's input do not work proper.

Reply:
aite gotcha